sdk/python/kfp/cli/diagnose_me_cli.py

```diff
@@ -46,13 +49,12 @@ def diagnose_me(ctx, json, project_id, namespace):
       human_readable=False)
   for app in ['Google Cloud SDK', 'gsutil', 'kubectl']:
     if app not in local_env_gcloud_sdk.json_output:
-      print(
-          '%s is not installed, gcloud, gsutil and kubectl are required' % app,
-          'for this app to run. Please follow instructions at',
+      raise RuntimeError(
+          f'{app} is not installed, gcloud, gsutil and kubectl are required' +
```

Comment:
Since we've not formally dropped Python 3.5 yet in setup.py, can we keep the previous string formatting here?

f-string is only available since Python 3.6

Reply:
@numerology Fixed it.